package.json

```diff
@@ -18,6 +18,9 @@
     "start": "react-scripts start",
     "build": "yarn verify"
   },
+  "pre-commit": [
+    "verify"
+  ],
```

Comment:
This new config tells the project to run the verify script before committing and if it has errors it doesn't commit without the `--no-verify` flag passed.